TAU release 1.0.25

TomaszLukawskiSam opened the pull request:
Dedicated platform: Tizen 4.0, Tizen 5.0

New features in 1.0.0:

- support for bezel-less devices
- extended widget control by touch
- new Date/Time pickers

Changes in version 1.0.25:
- Removing `dist` folder from repository
- (SVACE) SwipeList: move animation function out of IIFE
- (SVACE) SearchBar: remove label element
- (SVACE) SearchBar: remove unecessary condition
- Remove code blocks that are never executed
- (SVACE) SnapListview: move animation function out of requestAnimationFrame
- (SVACE) Slider: added "return" to "_setValue" method
- (SVACE) ToggleSwitch.extra: remove unreachable code
- (SVACE) Datetimepicker: omit number conversion to string
- (SVACE) FastScroll: removed unreached parts of code
- (SVACE) performance: add return at the end of get function
- (SVACE) Datetimepicker: Remove unnecessary return
- Sync jQuery library version (#534)
- (SVACE) RotaryEventBinder: removing name from animation function
- (SVACE) Scroller: removing redundant assignment
- (SVACE) PageIndicator: the option "intervalAngle" must be a number
- (SVACE) BaseWidget: added "return" at the end of "trigger" methods
- (SVACE) support: remove redundant assignment for pointerEvents
- (SVACE) history manager: fix for expression evaluated always true
- (SVACE) ArcListview: use "return" consistently
- (SVACE) vmouse: added 'return' on end of function
- (SVACE) engine: Refactoring of events handler to anonymous function
- (SVACE) Progress: define step function
- (SVACE) Scrollview: add return at the end of findPositionAnchor method
- (svace) ArcListview: Restore marquee animation on click
- BaseWidget: copy all style classes from element to _stateDOM
- (SVACE) engine.js: fix expression for element always evaluate to true
- (SVACE) Gruntfile: fix for unconsistently "return" use
- Make functions anonymous
- Make functions anonymous in BaseWidget.js
- Make functions anonymous in load.js
- Remove duplicated assignements
- Fix potential TypeError
- Move variable declarations before their usage
- Remove Math and Set overrides
- Move the declaration of "targetPath"
- Make functions anonymous
- Remove UIComponents_legacy
- Add missing I index in IndexScrollBar example
- Delete jira-watcher.js
- IndexScrollbar: fix for scroll to select index
- Enable scrolling for drawer content
- fixup! (UIComponents) SnapListView: fix for wrong selection on rotary scrolling
- ArcListview: do not show selection while scrolling
- ArcListview: fix showing selection on click while scrolling
- ArcListview: fix for selection highlight
- (UIComponents) SnapListView: fix for wrong selection on rotary scrolling
- Marquee: Deprecated value 'ellipsis' for option 'ellipsisEffect'
- UIComponents: fix for circle-helper for support bezel in popup content
- (util) rotaryScrolling: added new methods for lock/unclock of scrolling
- Marquee: fix setIteration
- Fix style error
- Marquee: fix infinite animation
- Marquee: allow multiple call to destroy() method
- Arclistview: fix showing highlight on slowly scroll
- Fix the problem that cannot build widget with one or none items (#540)
- Fix hiding header while scrolling on short lists
- Arclistview: ensure correct radio button selected after scroll
- Fix drawer example message
- Don't set selected month in DatePicker

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

TomaszLukawskiSam commented:
Now, TAU release is under verification.

hsiwkin commented:
We've performed tests of this release with @grzegorz-czajkowski  both on device and emulator and it looks good - we haven't found any issues.

grzegorz-czajkowski reviewed (approved):
No regression was found.

TomaszLukawskiSam committed:
TAU release 1.0.25

Dedicated platform: Tizen 4.0, Tizen 5.0

New features in 1.0.0:

- support for bezel-less devices
- extended widget control by touch
- new Date/Time pickers

Changes in version 1.0.25:
- Removing `dist` folder from repository
- (SVACE) SwipeList: move animation function out of IIFE
- (SVACE) SearchBar: remove label element
- (SVACE) SearchBar: remove unecessary condition
- Remove code blocks that are never executed
- (SVACE) SnapListview: move animation function out of requestAnimationFrame
- (SVACE) Slider: added "return" to "_setValue" method
- (SVACE) ToggleSwitch.extra: remove unreachable code
- (SVACE) Datetimepicker: omit number conversion to string
- (SVACE) FastScroll: removed unreached parts of code
- (SVACE) performance: add return at the end of get function
- (SVACE) Datetimepicker: Remove unnecessary return
- Sync jQuery library version (#534)
- (SVACE) RotaryEventBinder: removing name from animation function
- (SVACE) Scroller: removing redundant assignment
- (SVACE) PageIndicator: the option "intervalAngle" must be a number
- (SVACE) BaseWidget: added "return" at the end of "trigger" methods
- (SVACE) support: remove redundant assignment for pointerEvents
- (SVACE) history manager: fix for expression evaluated always true
- (SVACE) ArcListview: use "return" consistently
- (SVACE) vmouse: added 'return' on end of function
- (SVACE) engine: Refactoring of events handler to anonymous function
- (SVACE) Progress: define step function
- (SVACE) Scrollview: add return at the end of findPositionAnchor method
- (svace) ArcListview: Restore marquee animation on click
- BaseWidget: copy all style classes from element to _stateDOM
- (SVACE) engine.js: fix expression for element always evaluate to true
- (SVACE) Gruntfile: fix for unconsistently "return" use
- Make functions anonymous
- Make functions anonymous in BaseWidget.js
- Make functions anonymous in load.js
- Remove duplicated assignements
- Fix potential TypeError
- Move variable declarations before their usage
- Remove Math and Set overrides
- Move the declaration of "targetPath"
- Make functions anonymous
- Remove UIComponents_legacy
- Add missing I index in IndexScrollBar example
- Delete jira-watcher.js
- IndexScrollbar: fix for scroll to select index
- Enable scrolling for drawer content
- fixup! (UIComponents) SnapListView: fix for wrong selection on rotary scrolling
- ArcListview: do not show selection while scrolling
- ArcListview: fix showing selection on click while scrolling
- ArcListview: fix for selection highlight
- (UIComponents) SnapListView: fix for wrong selection on rotary scrolling
- Marquee: Deprecated value 'ellipsis' for option 'ellipsisEffect'
- UIComponents: fix for circle-helper for support bezel in popup content
- (util) rotaryScrolling: added new methods for lock/unclock of scrolling
- Marquee: fix setIteration
- Fix style error
- Marquee: fix infinite animation
- Marquee: allow multiple call to destroy() method
- Arclistview: fix showing highlight on slowly scroll
- Fix the problem that cannot build widget with one or none items (#540)
- Fix hiding header while scrolling on short lists
- Arclistview: ensure correct radio button selected after scroll
- Fix drawer example message
- Don't set selected month in DatePicker

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>
